themes/custom/az_barrio/az_barrio.theme

```diff
@@ -539,12 +539,42 @@ function az_barrio_theme_suggestions_menu_alter(array &$suggestions, array $vari
 }
 
 /**
- *  Implements template_preprocess_block().
+ * Implements hook_preprocess_HOOK() for breadcrumb templates.
+ */
+function az_barrio_preprocess_breadcrumb(&$variables) {
+  if (\Drupal::moduleHandler()->moduleExists('easy_breadcrumb')) {
+    $add_home_class = FALSE;
+    $config = \Drupal::config('easy_breadcrumb.settings');
+    if ($config->get('include_home_segment')) {
+      if (\Drupal::service('path.matcher')->isFrontPage()) {
+        if ($config->get('hide_single_home_item')) {
+          $add_home_class = TRUE;
+        }
+      }
+      else {
+        $add_home_class = TRUE;
+      }
+    }
+
+    foreach ($variables['breadcrumb'] as $key => &$item) {
+      $item['classes'] = 'breadcrumb-item';
+      if ($add_home_class && $key === 0) {
+        $item['classes'] .= ' breadcrumb-item-home';
```

Comment:
This needs to be a string?

Reply:
The breadcrumb template provided by core doesn't have an existing classes or attributes variable.